Makefile.am

```diff
@@ -240,15 +241,20 @@ BUILT_SOURCES = .version git-version.h
 CLEANFILES = crun.spec .version git-version.h $(LUACRUN_ROCKSPEC)
 
 if ENABLE_CRUN
-man1_MANS = crun.1
+man1_MANS = crun.1 krun.1
 endif
 
 crun.1: $(abs_srcdir)/crun.1.md
 if HAVE_MD2MAN
 	$(MD2MAN) -in $(abs_srcdir)/crun.1.md -out crun.1
 endif HAVE_MD2MAN
 
-generate-man: crun.1
+krun.1: $(abs_srcdir)/krun.1.md
+if HAVE_MD2MAN
+	$(MD2MAN) -in $(abs_srcdir)/krun.1.md -out krun.1
+endif HAVE_MD2MAN
+
+generate-man: crun.1 krun.1
```

Comment:
if you're making the md and final man page, do we need md2man?

Reply:
@giuseppe does not want to require md2man to build.